build/BUILD.wpt

```diff
@@ -1,8 +1,17 @@
-filegroup(
-    name = "url",
-    srcs = glob(
-        include = ["url/**/*"],
-        allow_empty = True,
+directories = glob(
+    ["*"],
+    exclude = glob(
+        [
+            "*",
+            ".*",
```

Comment:
Does this ignore `.github` folder? Can you validate?

Reply:
I tweaked the globs a little to get rid of it.